### DISCUSSION TITLE
problem accessing http://localhost:3000

### DISCUSSION THREAD

**Discussion by mmuhammadeuss** [open-webui/open-webui#833] [General]
Hi TEAM

I use Windows 10 and here are the actions taken:
1.Download and install Docker Desktop from Docker's official website: **ok**
2.Download the latest version from https://ollama.com/: **ok**
3.After installing Ollama, check if it's working by visiting http://127.0.0.1:11434/ in web browser: **ok**
4.Download the docker open-webui image: **ok**
5.Start a docker container based on the open-webui image
docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/ open-webui/open-webui:main: **ok**
6. Access via http://localhost:3000: **problem at this level because I cannot access the web interface**
error message: Connection failed Firefox cannot establish a connection with the server at localhost:3000

how to resolve this?

**Comment by jannikstdl** [comment]
Can you access ollama under ` http://localhost:11434/ ` ?

Try port forwarding on another port like `docker run -d -p 3333:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

**Reply by RaphyTwin** [reply]
Does this make it public for everyone in the pubic internet?

**Comment by jannikstdl** [comment]
@tjbck works as espected with the latest version on my machine

**Comment by bunt-oxide** [comment]
It was my firewall.

**Comment by mmuhammadeuss** [comment]
@jannikstdl thank you for your support
1. ollama under http://localhost:11434 : **ok**
2. ollama under http://127.0.0.1:11434 : **ok**
3. Try port forwarding on another port like 3333: **same problem, cannot access**
error message: Connection failed Firefox cannot establish a connection with the server at localhost:3333

Note that I disabled my firewall

please how to resolve this?

**Reply by IOrlandoni** [reply]
hello

**Comment by bunt-oxide** [comment]
Take a look at your log, you might find the cause.

> Docker Desktop > Containers > open-webui

**Reply by salpelter** [reply]
This helped. I was missing some package apparently. The log even told me how to install it (using pip). Restarted the container after that and it worked!

**Comment by mmuhammadeuss** [comment]
Based on my machine's resources and the logs, it took around ten minutes for the application to fully set up. I now access the application. thank you for your support

**Reply by praktix1234** [reply]
i was stuck at the same point. if you are behind a bad internet connection, the start up of the container takes up to 15 minutes on the first time. its loading 30 files but the Docker Log outputs it very late.

**Comment by TZECHIN6** [comment]
Hi sir, have you solved this? I cannot access the localhost:8080 port (as i have set --network=host). How you manage to fix it.

**Comment by nickGermi** [comment]
I deleted existing docker container using the Docker UI, then tried following command:
`docker run -d --name open-webui -p 3000:8080 -v open-webui-data:/app/data --pull=always ghcr.io/open-webui/open-webui:main`

Now http://localhost:3000 works
